packages/core/src/app.ts

```diff
@@ -477,7 +477,7 @@ export class App {
      * Locks the app and wipes all sensitive information from memory.
      */
     async lock() {
-        [this.account!, ...this.state.orgs, ...this.state.vaults].forEach((each) => each.lock());
+        [this.account!, ...this.state.orgs, ...this.state.vaults].forEach((each) => each && each.lock());
```

Comment:
This was the error it found. Basically some `each` was undefined or null, can't remember now, when locking an account without any items.

Reply:
Good catch! Btw, you can write this just a tiny bit more concisely using the optional chaining syntax:
```ts
each?.lock()
```

Reply:
Good suggestion!